src/promql/src/planner.rs

```diff
@@ -139,14 +141,56 @@ impl<S: ContextProvider> PromPlanner<S> {
                 name: "Prom Unary Expr",
             }
             .fail()?,
-            PromExpr::Binary(PromBinaryExpr { lhs, rhs, .. }) => {
-                let _left_input = self.prom_expr_to_plan(*lhs.clone())?;
-                let _right_input = self.prom_expr_to_plan(*rhs.clone())?;
-
-                UnsupportedExprSnafu {
-                    name: "Prom Binary Expr",
+            PromExpr::Binary(PromBinaryExpr { lhs, rhs, op, .. }) => {
+                match (
+                    Self::try_build_literal_expr(lhs),
+                    Self::try_build_literal_expr(rhs),
+                ) {
+                    (Some(_lhs), Some(_rhs)) => {
+                        todo!("is this a legal case?")
```

Comment:
Is this a binary expr between two literals? Looks like promql supports this

https://github.com/prometheus/prometheus/blob/6dcfb717403e4a58c770dad796a09e9570a7a684/promql/testdata/literals.test#L10

Reply:
I don't mean the expression itself. I'm considering if this case should be captured before this match. As the literal-only expression is not a plan. I'll change this to a todo comment and return an unsupported error for now.